### ISSUE TITLE
Updated Marauder flasher you may like!

### ISSUE THREAD

**Issue by UberGuidoZ** [DrB0rk/Flipper-Zero-Boards#8]
Thanks for your work on the multiboards!

Saw the flasher you have and it is super old code. [Take a peek at this](https://github.com/UberGuidoZ/Flipper/tree/main/Wifi_DevBoard/FZ_Marauder_Flasher) and see if it works better for you. <3

**Comment by DrB0rk** [comment]
Will do, thanks! 
Quickly threw that together a while ago :)

**Comment by Slicc98** [comment]
Will this work for the multiboards?

**Comment by FalsePhilosopher** [comment]
> Will this work for the multiboards?

Why would they suggest it if it didn't work?

**Comment by UberGuidoZ** [comment]
Yes, it will work for all the boards. (And has been updated multiple times since too.)

**Comment by jaochoo** [comment]
So, I have purchased the multiboard v2 and there the board does not have any USB port. I am wondering how I flash the board. Please do help me.
